### PR TITLE
App configuration and docker

### LINKED ISSUE #1: Create Docker image and publish it in a CI pipeline
**Is your feature request related to a problem? Please describe.**
When new changes are added to the main branch of the oxygen app, a new Docker image should be built and published on DockerHub

**Describe the solution you'd like**
A new Docker image should be published to DockerHub when changes are pushed in the main branch of the oxygen app.

**Describe alternatives you've considered**
The only alternative is to create and publish manually

**Additional context**
The docker image should run the metrics flask app on execution (including connecting to the database)

### LINKED ISSUE #7: Configure the application
**Is your feature request related to a problem? Please describe.**
We need to configure the application with environment variables. The solution needs to have tests.

**Describe the solution you'd like**
We want a file that contains the environment variable to configure the application.

**Describe alternatives you've considered**
No alternatives.

**Additional context**
If a environment variable is not defined, we need to set a default value. If the token is not set we need to return an error. For each modification in the code we need to add unit tests.

### PR DESCRIPTION
**Pull request by chargag**
# Description

Includes docker configuration files to allow for image building and publishing. To allow this, the application was also updated to allow for environment variables.

Fixes #1 and Fixes #7 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

A unit test was added to validate that environment variables are correctly used and interpreted by the application.

- [x] test_env_vars

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,6 @@
+HOST='http://34.95.34.5'
+TOKEN='O3WhZRXLCf'
+DATABASE=
+TICKETS=3
+T_MAX=90
```
Comment (alex6ripetti) [inline .env:5]: Pour les températures, il faudrait mettre des températures qu'on voudra dans une entreprise, genre de 19° à 25° par exemple, pas les extrêmes

File: docker.env [hunk 2]
```diff
@@ -0,0 +1,6 @@
+HOST='http://34.95.34.5'
+TOKEN='O3WhZRXLCf'
+DATABASE=
+TICKETS=3
+T_MAX=90
```
Comment (alex6ripetti) [inline docker.env:5]: Même commentaire que pour le .env classique

**Review by Balnian — COMMENTED** [review]
LGTM, juste faire les changement recommander par @alex6ripetti

**Review by Balnian — APPROVED** [review]
Everything is good, you can merge

**Pull request merged**